examples/keras_io/tensorflow/vision/mirnet.py

```diff
@@ -0,0 +1,505 @@
+"""
+Title: Low-light image enhancement using MIRNet
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Converted to Keras Core by: [Soumik Rakshit](http://github.com/soumik12345)
+Date created: 2021/09/11
+Last modified: 2021/09/15
```

Comment:
Update date

Reply:
Should `Date created` be updated as well?

Reply:
No, we can keep the original one